en_ewt-ud-train.conllu

```diff
@@ -150895,7 +150895,7 @@
 2	you	you	PRON	PRP	Case=Nom|Person=2|PronType=Prs	3	nsubj	3:nsubj	_
 3	knew	know	VERB	VBD	Mood=Ind|Number=Sing|Person=2|Tense=Past|VerbForm=Fin	18	advcl	18:advcl:if	_
 4	what	what	DET	WDT	PronType=Int	5	det	5:det	_
-5	kind	kind	NOUN	NN	Number=Sing	11	nsubj	11:nsubj	_
+5	kind	kind	NOUN	NN	Number=Sing	11	obj	11:obj	_
```

Comment:
Not sure about the tree structure. Should the head be "knew"?

Reply:
Yes, I think "knew" is the local head...
And I think the whole sentence can be rephrased as "If you knew they put what kind of meat in canned cat food you'd understand this", so "kind" is `obj` of "they"

Reply:
OK so the head of "kind" should be 3 rather than 11 right?

Reply:
"you knew [what kind of meat they put in cat food]"—your analysis is `ccomp(knew, put)`, `obj(put, kind)`.

But is this a free relative clause such that it should be `obj(knew, kind)`, `acl:relcl(kind, put)`?

I can never remember the rules for this. :( @amir-zeldes

Reply:
I think "kind" is `obj` of "put", since the sentence could be rephrased as "they put what kind of meat in canned cat food" . So "kind" is `11:obj`.

Reply:
But it could be a relative clause extracted from object. cf. "the kind of meat that they put in cat food"

Reply:
Yup, I think it's a free relative:

  * You knew the kind
  * You knew the kind (that it was)
  * You knew that kind, which they put in cat food

Therefore:
    * obj(knew, kind)
    * det(kind, what)
    * acl:relcl(kind, put)

And in edeps you can have the secondary dependency (put ->obj kind)